test/jdk/java/net/httpclient/ShortResponseBody.java

```diff
@@ -372,82 +343,6 @@ public int read(byte[] buf, int offset, int length) {
     static final List<String> MSGS_ORDER =
             List.of("no bytes", "status line", "header");
```

Comment:
Original change removed this. Looks unused.

Reply:
Thanks for spotting this, fixed.